src/components/Notification/Readme.md

```diff
@@ -1,3 +1,7 @@
+```js static
+import Notification from "openseadragon-react-viewer";
```

Comment:
These smaller imports should be wrapped in brackets... ie. `import { Notification } from "openseadragon-react-viewer"` I think.